qlty-check/src/planner/config_files.rs

```diff
@@ -15,6 +16,27 @@ pub struct PluginConfigFile {
     pub contents: String,
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct ConfigStagingOperation {
+    pub source_path: PathBuf,
+    pub destination_path: PathBuf,
+    pub operation_type: ConfigOperationType,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub enum ConfigOperationType {
+    /// Copy config from repository to staging area
+    CopyToStagingArea,
+    /// Copy config from repository to workspace root
+    CopyToWorkspaceRoot,
```

Comment:
Is there ever a case where we want to `CopyToStagingArea` but we don't want to `CopyToWorkspaceRoot` or vice versa? In other words, are these two operation types always going to be used together?